### PR TITLE
Build infr docker containers

### PR DESCRIPTION
**Pull request by JayH5**
- No app yet
- Add node dependencies from package.json file
- Only cpython for now
- Remove a lot of options from vumi-go-dockerbuild to get things
  going

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
```
Comment (JayH5) [inline package.json:1]: Not sure of best way to do this... these packages are from [here](https://github.com/praekelt/mama-ng-deploy/blob/develop/docker-vms/jssandbox/Dockerfile#L8).

The versions (where possible) are from [here](https://github.com/praekelt/vumi-go/blob/develop/package.json#L11-L23).

**Comment by JayH5** [comment]
Ready for review. This doesn't actually build anything from a repo yet, just pip installs.

For a list of "infr" images see the [mama-ng-deploy readme](https://github.com/praekelt/mama-ng-deploy/blob/develop/README.md). I haven't included things that we are taking out of Docker containers. All the existing Dockerfiles are in the mama-ng-deploy repo [here](https://github.com/praekelt/mama-ng-deploy/tree/develop/docker-vms).

File: package.json (hunk 1 above)
Comment (jerith) [inline package.json:1]: Maybe give them all `"*"` versions?

**Comment by jerith** [comment]
:+1:

**Pull request merged**